packages/manager/apps/telecom/src/app/telecom/telephony/line/line-main.html

```diff
@@ -21,7 +21,7 @@
 
         <oui-message
             data-type="warning"
-            data-ng-if="LineCtrl.displayMgcpBanner"
+            data-ng-if="LineCtrl.isMgcpBannerAvailable() && LineCtrl.displayMgcpBanner"
```

Comment:
As conditions are linked, you should set `displayMgcpBanner` only if `isMgcpBannerAvailable()` is `true`

Reply:
Indeed